HeterogeneousCore/SonicTriton/plugins/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<library   name="HeterogeneousCoreSonicTritonPlugins_converters" file="converters/*.cc">
+  <use   name="FWCore/Utilities"/>
+  <use   name="FWCore/ParameterSet"/>
+  <use   name="FWCore/PluginManager"/>
+  <use   name="DataFormats/Common"/>
+  <use   name="HeterogeneousCore/SonicTriton"/>
```

Comment:
I think this is really the only CMSSW include needed in the plugin BuildFile (the ones from the package BuildFile don't need to be repeated)